episodes/introduction.md

```diff
@@ -89,10 +179,24 @@ All of the above are things a Hub Champion should do to enable their community t
 
 ::::
 
+## Next Steps
+
+The next part of the training is a self-guided study of the following episodes
+
+-. {ref}`episode-admin`
+-. {ref}`episode-images`
+-. {ref}`episode-data`.
+
+and concludes with an in-person episode covering {ref}`episode-troubleshoot`.
```

Comment:
I don't think "in-person" is the right wording here. Maybe "a synchronous session"?